SU2_CFD/include/fluid/CConstantDiffusivity.hpp

```diff
@@ -0,0 +1,58 @@
+/*!
+ * \file CConstantDiffusivity.hpp
+ * \brief Defines constant mass diffusivity.
+ * \author S. Vitale, M. Pini, G. Gori, A. Guardone, P. Colonna, T. Economon
```

Comment:
Clang format this file please, and  maybe the authors need to be updated?

Reply:
OK, I will do it. Thank you so much Pedro for the comments.

Reply:
This was originally Tom's implementation, so put Tom and Cristopher as authors.